spk/sabnzbd-beta/Makefile

```diff
@@ -0,0 +1,58 @@
+SPK_NAME = sabnzbd-beta
+SPK_VERS = 0.8.0
+SPK_REV = Alpha3
+SPK_ICON = src/sabnzbd-beta.png
+DSM_UI_DIR = app
+BETA = 1
+
+DEPENDS  = cross/busybox cross/par2cmdline cross/unrar cross/$(SPK_NAME)
+WHEELS = six configobj
+SPK_DEPENDS = "python>=2.7.6-8"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = SABnzbd makes Usenet as simple and streamlined as possible by automating everything we can. All you have to do is add an .nzb. SABnzbd takes over from there, where it will be automatically downloaded, verified, repaired, extracted and filed away with zero human interaction.
+DESCRIPTION_FRE = SABnzbd rend Usenet aussi simple et automatisé que possible. Vous n\\\'avez qu\\\'a ajouter un .nzb, SABnzbd s\\\'occupera du reste sans aucune intervention de votre part : téléchargement, vérification, réparation, extraction et déplacement.
+DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatizando todo lo que se puede. Todo lo que tienes que hacer es agregar un archivo .nzb. SABnzbd empieza desde ahí. Tus archivos serán automáticamente descargados, verificados, reparados, descomprimidos y archivados.
+ADMIN_PORT = 8080
```

Comment:
This should be changed to a different port, otherwise the consequence is that the sabnzbd package and the sabnzbd-testing package can't be installed simultaneously. See https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports for instructions on how to choose a port.